src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php

```diff
@@ -86,7 +86,7 @@ public function configureOptions(OptionsResolver $resolver)
                 'window.dropImages' => $this->trans('Drop images here', 'Admin.Catalog.Feature'),
                 'window.selectFiles' => $this->trans('or select files', 'Admin.Catalog.Feature'),
                 'window.recommendedSize' => $this->trans('Recommended size 800 x 800px for default theme.', 'Admin.Catalog.Feature'),
-                'window.recommendedFormats' => $this->trans('JPG, GIF or PNG format.', 'Admin.Catalog.Feature'),
+                'window.recommendedFormats' => $this->trans('JPG, GIF, PNG or WEBP format.', 'Admin.Catalog.Feature'),
```

Comment:
```suggestion
                'window.recommendedFormats' => $this->trans('JPG, GIF, PNG or WebP format.', 'Admin.Catalog.Feature'),
```

Reply:
Maybe it's a nitpick, but eventually, it is WebP, not WEBP. What do you think @PrestaEdit ?